app/config/AppConfig.scala

```diff
@@ -35,40 +35,42 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   lazy val customsFinancialsApi: String = servicesConfig.baseUrl("customs-financials-api") +
     config.getOptional[String]("customs-financials-api.context").getOrElse("/customs-financials-api")
 
-  lazy val customsDataStore: String = s"${servicesConfig.baseUrl("customs-data-store")}${
-    config.getOptional[String](
-      "microservice.services.customs-data-store.context").getOrElse("/customs-data-store")
-  }"
+  private lazy val customsDataStoreBaseUrl    = servicesConfig.baseUrl("customs-data-store")
+  private lazy val customsDataStoreContextUrl =
+    config.getOptional[String]("microservice.services.customs-data-store.context").getOrElse("/customs-data-store")
+  lazy val customsDataStore: String           =
```

Comment:
Better to give a line break before this line to make it more readable and to differentiate from private vals